### ISSUE TITLE
Apple: plantscan3d doesn't open after following installation instruction

### ISSUE THREAD

**Issue by Micbut** [openalea/plantscan3d#16]
Hi,

I want to install PlantScan3d on my macbook 13-inch, M1, 2020 running with Ventura 13.0.1.
I followed the instruction on https://plantscan3d.readthedocs.io/en/latest/userguide/installation.html#anaconda
however once I try to run from Terminal it gives me the following error
<img width="904" alt="Screenshot 2023-01-11 at 10 04 41" src="https://user-images.githubusercontent.com/54619028/211764575-30e7ed59-3f49-4b0b-a6ff-0ed24d1b29d5.png">

**Comment by Micbut** [comment]
![Screenshot 2023-01-26 at 16 51 41](https://user-images.githubusercontent.com/54619028/214883196-b1e8666b-e455-4fe4-aeb3-6dd503f842f4.png)

**Comment by fredboudon** [comment]
I'm trying to build a new version. I keep you updated

**Comment by thomasarsouze** [comment]
A solution has been posted [here](https://fspm.discourse.group/t/installation-on-apple-silicon/216/5).
I copy it here for the sake of archiving:
```
CONDA_SUBDIR=osx-64 conda create -n PS3D python=3.10
conda activate PS3D
conda config --env --set subdir osx-64
conda install -c fredboudon -c conda-forge openalea.plantgl openalea.mtg pyqglviewer -y
conda install -c fredboudon -c conda-forge openalea.plantscan3d -y
```

While there is still no dedicated arm64 version proposed, this workaround should work.
Please @Micbut let us know if this is ok for you.

**Comment by Micbut** [comment]
it finally works!

I copy here what I wrote also in the other forum.

However, it was quite tricky to install, because I will getting errors when i was trying to start it.

Somehow, plantscan3d needs the dynamic library libjpeg.8.dylib and with the code you shared, version 9 gets installed. Furthermore, it needs the libjpeg.8.dylib x86-64 version and not the ARM one. So I had to manually install libjpeg.8.dylib using Rosetta2 to install it with x86-64 architecture, and it finally worked.

None of these changes would have been possible without the help of GPT-4
